lib/absinthe/schema/notation.ex

```diff
@@ -1069,10 +1085,11 @@ defmodule Absinthe.Schema.Notation do
     |> expand_ast(env)
     |> Keyword.update(:values, [], fn values ->
       Enum.map(values, fn ident ->
-        value_attrs = handle_enum_value_attrs(ident, module: env.module)
+        value_attrs = handle_enum_value_attrs(ident, [], env)
```

Comment:
bug fix: `handle_enum_value_attrs` needs an `env` struct to pass into `expand_ast` and this keyword list `module: env.module` to simulate the `env` doesn't work)